### PR TITLE
adding version history file for v1.18.3

### PR DESCRIPTION
**Pull request by ankatare**
Signed-off-by: Abhay Narayan Katare <abhay.katare@india.nec.com>

Commit Message: Adding Version History for v1.18.3
Additional Description:
Risk Level: LOW
Testing: 
Docs Changes: Yes
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

**Comment by jmarantz** [comment]
It's hard to suss this out of the logs currently but the problem in the format checker is trailing whitespace in the new .rst file.

**Comment by phlax** [comment]
> It's hard to suss this out of the logs currently but the problem in the format checker

ill add a message that you need to adjust your editor for this case (#17008)

**Comment by phlax** [comment]
@ankatare to make this work you will have to add `1.18.3` to the mapped versions in `docs/conf.py`

here: 

https://github.com/envoyproxy/envoy/blob/9d2b3ff9d41ab4469bd870e78cde2c27845fc8f5/docs/conf.py#L343-L347

**Comment by ankatare** [comment]
@phlax Ok. Thanks for help. let me check this.

File: docs/conf.py [hunk 1]
```diff
@@ -322,5 +322,6 @@ def setup(app):
     'v1.17.0': ('https://www.envoyproxy.io/docs/envoy/v1.17.0', None),
     'v1.17.1': ('https://www.envoyproxy.io/docs/envoy/v1.17.1', None),
     'v1.17.2': ('https://www.envoyproxy.io/docs/envoy/v1.17.2', None),
-    'v1.18.0': ('https://www.envoyproxy.io/docs/envoy/v1.18.2', None)
+    'v1.18.0': ('https://www.envoyproxy.io/docs/envoy/v1.18.2', None),
+    'v1.18.0': ('https://www.envoyproxy.io/docs/envoy/v1.18.3', None)
```
Comment (phlax) [inline docs/conf.py:326]: this needs to be `1.18.3` also if you want to refer to this version as such

Comment (ankatare) [reply docs/conf.py:326]: Ok.

**Comment by ankatare** [comment]
@phlax seems again issue with glint.. error as follows.. could you please advise some editior here to fix it

2021-06-18T16:19:11.0257679Z ##[error]: TESTS FAILED:
2021-06-18T16:19:11.0258368Z ##[error]:  main@ /source/ci/format_pre.sh :47 (glint)
2021-06-18T16:19:11.0258931Z ##[error]:      Please fix your editor to ensure:
2021-06-18T16:19:11.0260121Z ##[error]:        - no trailing whitespace
2021-06-18T16:19:11.0260851Z ##[error]:        - no mixed tabs/spaces
2021-06-18T16:19:11.0261554Z ##[error]:        - all files end with a newline
2021-06-18T16:19:12.1375137Z ##[error]Bash exited with code '1'.

**Comment by phlax** [comment]
> @phlax seems again issue with glint.. error as follows.. could you please advise some editior here to fix it

not sure @ankatare - it depends on your editor - i use emacs

the main thing is to fix it so that on save it:

- removes trailing whitespace
- ensure a newline
- prevents mixed tabs/space indentation

**Comment by ankatare** [comment]
@phlax Ok. let me check. Thanks

**Comment by ankatare** [comment]
@phlax would you please suggest any script to run and fix the build issues similar to what we have `ci/run_envoy_docker.sh 'ci/do_ci.sh check_format'`.

**Comment by htuch** [comment]
@ankatare `ci/do_ci.sh fix_format`?

**Comment by ankatare** [comment]
@htuch Ok. Let me check it locally

**Comment by ankatare** [comment]
@htuch i run `ci/run_envoy_docker.sh 'ci/do_ci.sh fix_format'` but it is not working for `ci/run_envoy_docker.sh 'ci/do_ci.sh format_pre'` , which is failing :(

**Comment by ankatare** [comment]
@htuch found fix . pushed changes :)

**Comment by ankatare** [comment]
@htuch @phlax PTAL

**Review by phlax — APPROVED** [review]
lgtm, thanks @ankatare

File: docs/root/version_history/v1.18.3.rst [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+---------
+*Changes expected to improve the state of the world and are unlikely to have negative effects*
+
+* zipkin: fix timestamp serializaiton in annotations. A prior bug fix exposed an issue with timestamps being serialized as strings.
+
+Removed Config or Runtime
+-------------------------
+*Normally occurs at the end of the* :ref:`deprecation period <deprecated>`
+
+* tls: removed `envoy.reloadable_features.tls_use_io_handle_bio` runtime guard and legacy code path.
```
Comment (mattklein123) [inline docs/root/version_history/v1.18.3.rst:22]: I doubt we removed this in a point release?

/wait

Comment (ankatare) [reply docs/root/version_history/v1.18.3.rst:22]: @mattklein123 Thnaks for review. i followed below path
https://www.envoyproxy.io/docs/envoy/v1.18.3/version_history/current
please suggest...

Comment (mattklein123) [reply docs/root/version_history/v1.18.3.rst:22]: You can check the release, but this must be a bug. We would never remove a runtime guard in a point release.

Comment (ankatare) [reply docs/root/version_history/v1.18.3.rst:22]: Are you saying https://www.envoyproxy.io/docs/envoy/v1.18.3/version_history/current also not Correct and `envoy.reloadable_features.tls_use_io_handle_bio `should be under "Bug Fixes" Section in version_history file for v1.18.3 ?

Comment (mattklein123) [reply docs/root/version_history/v1.18.3.rst:22]: I'm saying that I don't think this change was in that release, but please check.

Comment (ankatare) [reply docs/root/version_history/v1.18.3.rst:22]: Ok. Would you please suggest where should i double check as i have only file https://www.envoyproxy.io/docs/envoy/v1.18.3/version_history/current to refer

Comment (mattklein123) [reply docs/root/version_history/v1.18.3.rst:22]: I would look at the branches: https://github.com/envoyproxy/envoy/tree/release/v1.18

Comment (ankatare) [reply docs/root/version_history/v1.18.3.rst:22]: humm.. Thanks for this .. i checked file https://github.com/envoyproxy/envoy/blob/v1.18.3/docs/root/version_history/current.rst 
and found same entry there as well. please suggest ...

**Comment by ankatare** [comment]
@phlax need your help on this as per @mattklein123 seems it is a doc bug... my point is from where i should collect exact data for v1.18.3 release ? as i see same data on https://github.com/envoyproxy/envoy/blob/v1.18.3/docs/root/version_history/current.rst
and https://www.envoyproxy.io/docs/envoy/v1.18.3/version_history/current

**Comment by phlax** [comment]
i think we can, lets close

**Pull request closed without merging**